### PR TITLE
Add generators start endpoint

### PR DESCRIPTION
**Pull request by shriramshankar**

### CODE REVIEW

File: api/v1/swagger.yaml [hunk 1]
```diff
@@ -11267,8 +11267,9 @@ definitions:
           The ipAddress where the collector is installed.
         maxLength: 60
       collectorConfig:
-        type: object
         $ref: "#/definitions/CollectorConfigResponse"
+      generatersAdded:
```
Comment (iamigo) [inline api/v1/swagger.yaml]: "generatersAdded" should be "generatorsAdded"

File: tests/api/v1/collectors/get.js [hunk 2]
```diff
@@ -75,7 +75,7 @@ describe('tests/api/v1/collectors/get.js >', () => {
   });
 
   it('get with limit ok', (done) => {
-    const c2 = u.toCreate;
+    const c2 = JSON.parse(JSON.stringify(u.toCreate));
```
Comment (iamigo) [inline tests/api/v1/collectors/get.js]: I think it's cleaner to put the parse+stringify logic in the utils module in a "getCollectorToCreate()" fn--that way every test which uses it doesn't have to repeat the parse+stringify pattern

File: tests/db/model/collector/create.js [hunk 3]
```diff
@@ -19,11 +19,11 @@ describe('tests/db/model/collector/create.js >', () => {
   let userId;
   let collectorObj;
   beforeEach((done) => {
+    collectorObj = JSON.parse(JSON.stringify(u.getCollectorObj()));
```
Comment (iamigo) [inline tests/db/model/collector/create.js]: same as comment above

File: tests/db/model/collector/find.js [hunk 4]
```diff
@@ -56,8 +88,32 @@ describe('tests/db/model/collector/find.js >', () => {
     .catch(done);
   });
 
+  it('Collector Instnace with related generators', (done) => {
```
Comment (jgraff2) [inline tests/db/model/collector/find.js]: typo

File: tests/db/model/collector/find.js [hunk 5]
```diff
@@ -56,8 +88,32 @@ describe('tests/db/model/collector/find.js >', () => {
...
+      const sg2 = generators.filter((gen) =>
+        gen.dataValues.name === generator2.name)[0].dataValues;
+      expect(sg1.id).to.include(generator1.id);
+      expect(sg2.id).to.include(generator2.id);
+      done();
+    })
+    .catch(done);
+  });
+
+  it('Collector Instnace without generators ', (done) => {
```
Comment (jgraff2) [inline tests/db/model/collector/find.js]: typo

**Pull request merged**